ktor-monitoring/build.gradle.kts

```diff
@@ -42,27 +41,50 @@ dependencies {
     testImplementation("io.ktor:ktor-server-tests:$ktor_version")
     testImplementation("org.jetbrains.kotlin:kotlin-test:$kotlin_version")
 }
+
 repositories {
     mavenCentral()
 }
 
-tasks.test {
-    useJUnitPlatform()
-    testLogging {
-        showExceptions = true
-        showCauses = true
-        showStackTraces = true
-        exceptionFormat = TestExceptionFormat.FULL
+tasks {
+    compileKotlin {
+        kotlinOptions {
+            jvmTarget = java_version
 
-        events(TestLogEvent.FAILED, TestLogEvent.SKIPPED, TestLogEvent.PASSED)
+            apiVersion = "1.6"
```

Comment:
what do you think about such cases use variables ? numbers are repeating

Reply:
ok